monkey/infection_monkey/network/info.py

```diff
@@ -35,60 +38,72 @@ class NetworkAddress:
     domain: Optional[str]
 
 
-if is_windows_os():
-
-    def get_routes():
+def get_routes() -> List[Tuple[int, int, str, bytes, str]]:
```

Comment:
Can we add a docstring? Also, it would be nice to have a NamedTuple or dataclass instead of a tuple

Reply:
+1